### ISSUE TITLE
update the SoLoader SDK to version 0.10.4 or higher

### ISSUE THREAD

**Issue by nicolas-raoul** [commons-app/apps-android-commons#5347]
I just saw this from the Play console:

> Your version of the SoLoader SDK is causing your app to crash for some users
>
> Your app uses a version of the SoLoader SDK that is causing crashes on 64 bit-only devices. To resolve this, update the SoLoader SDK to version 0.10.4 or higher.

I also see this in `app/build.gradle`:

     testImplementation 'com.facebook.soloader:soloader:0.10.1'

**Comment by Jasson-Hsu** [comment]
Hi @nicolas-raoul,
It seems easy to fix this issue by changing the SoLoader SDK version in dependencies.

Did I understand this issue correctly?
If so can you assign this issue to me?

Thanks

**Comment by nicolas-raoul** [comment]
Yes potentially easy, just meeds some testing afterwards. 🙂 It is yours!

**Comment by Jasson-Hsu** [comment]
Hi @nicolas-raoul, sorry for updating late for such an easy issue.

I changed the dependencies to `testImplementation "com.facebook.soloader:soloader:0.10.5"`

But when I ran the unit tests,  ` fun testCreateCameraPictureFile()` failed caused by 
`java.lang.IllegalArgumentException: Failed to find configured root that contains C:\Users\Jasso\AppData\Local\Temp\robolectric-Method_testCreateCameraPictureFile17231003190764080051\fr.free.nrw.commons.beta-dataDir\cache\CommonsContributions\e6115ab3-969e-4ac7-aeda-a83b559a80267402240706843156903.jpg`

But it seems caused by my local environment and OS system.
Can I open a pull request to run the CI test?
